ontology/yaml/resources/states/states.yaml

```diff
@@ -18,9 +18,11 @@ AUTO: Running under automatic control.
 MANUAL: "Running under manual (hand) control."
 OPEN: Open position, typically for a valve or other pass-though.
 CLOSED: Closed position, typically for a valve or other pass-though.
+VERY_LOW : Very low speed or output setting.
```

Comment:
This is not an acceptable state because "VERY" is subjective. Can you please elaborate on the total number of discrete speed states this FCU has.